src/domain/entities/note.ts

```diff
@@ -10,6 +11,21 @@ export type NoteInternalId = number;
  */
 export type NotePublicId = string;
 
+/**
+ * Reference to a particular tools that were used for note creation
+ */
+export type ToolUsedInNoteContent = {
+  /**
+   * Name of certain editor tool
+   */
+  name: EditorTool['name'],
+
+  /**
+   * Id of certain editor tool (nanoid)
+   */
+  id: EditorTool['id']
+}[];
```

Comment:
```suggestion
};
```

lets make it a singular form and use like `ToolUsedInNoteContent[]` where needed